src/mozanalysis/frequentist_stats/linear_models/functions.py

```diff
@@ -194,23 +194,60 @@ def _extract_absolute_uplifts(
     - output (pd.Series): the set of inferences. See `_make_joint_output`.
     """
     output = _make_joint_output(alphas, "abs_uplift")
-
     parameter_name = f"C(branch, Treatment(reference='{ref_branch}'))[T.{branch}]"
-
     output.loc[("abs_uplift", "0.5")] = results.params[parameter_name]
     output.loc[("abs_uplift", "exp")] = results.params[parameter_name]
 
+    # i = np.where(results.params.index == parameter_name)[0][0]
     for alpha in alphas:
-        lower, upper = results.conf_int(alpha=alpha).loc[parameter_name]
-        low_str, high_str = stringify_alpha(alpha)
+        ci = results.conf_int(alpha=alpha)
+        lower, upper = ci.loc[parameter_name]
+        low_str, high_str = _stringify_alpha(alpha)
         output.loc[("abs_uplift", low_str)] = lower
         output.loc[("abs_uplift", high_str)] = upper
-
     return output
 
 
+def _create_datagrid(
```

Comment:
Have you tested to see how often conclusions from inferences using the data grid matches when the full data is used? Do you think it's worthwhile to give users the option to use the full data, rather than the approximation, when the data is small enough to easily fit in memory?

Reply:
Ah, I see the unit test comparing the results to the relative uplifts from bootstrapping. That accomplishes this, correct?

Reply:
So comparing the relative uplifts to bootstrapping does check for this in a roundabout way, but it's probably valuable to add a more precise test. 

As far as using the option to use the full data... I think that could be a helpful extension, but I'm wondering if we can consider it out of scope for this PR and add it to the backlog?

Reply:
2 test cases added

Reply:
I agree re: potentially adding the feature later.